tikv-configuration-file.md

```diff
@@ -752,6 +749,7 @@ raftstore 相关的配置项。
 + 一轮处理数据落盘的最大请求个数。
 + 默认值：256
 + 最小值：大于 0
++ 最大值: 10240
```

Comment:
16384

Reply:
@glorv Yes, I asked before, users didn't use 5 digits, so 10240 is big enough, later use 16384 for 2^ (from PR reviewer's advice)